packages/constellation/postcss.config.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable @typescript-eslint/no-var-requires, import/no-extraneous-dependencies */
+/* eslint-disable  */
+const cssnano = require('cssnano')
+
+module.exports = {
+  plugins: [
+    require('tailwindcss'),
```

Comment:
Does this still work? I believe the newer formatting for this file is like this:
```
module.exports = {
  plugins: {
    tailwindcss: {},
    autoprefixer: {},
  }
}
```
https://tailwindcss.com/docs/installation/using-postcss

Reply:
I'll take a look to see if I can accomplish the same thing with the newer format.